### PR TITLE
Environment dropdown

### PR DESCRIPTION
**Pull request by ninabreznik**
Move Environment dropdown in Contract tab on top

screen shot 2017-04-24 at 12 30 49 am

### CODE REVIEW

**Comment by yann300** [comment]
perhaps some possible improvement: 
 - `select exe env` and the dropdown list on the same line.
 - `Transaction gas limit` on the left.
 - `Value 'e.g' .... ` on the left.
 - make padding/margin uniform.

**Comment by chriseth** [comment]
Perhaps: `Value [.......] (e.g. .7 ether ...)`, i.e. the input box in between.

File: index.html [hunk 1]
```diff
@@ -87,6 +87,30 @@
...
+						</option>
+						<option id="web3-mode"
+						title="Execution environment connects to node at localhost (or via IPC if available), transactions will be sent to the network and can cause loss of money or worse!
+						If this page is served via https and you access your node via http, it might not work. In this case, try cloning the repository and serving it via http."
+						value="web3"
+						name="executionContext">
+						Web3 Provider
+					</option>
+				</select>
+			</span>
```
Comment (chriseth) [inline index.html]: Indentation looks weird.

**Comment by ninabreznik** [comment]
@chriseth @yann300 I will create a new pull request, so we can close this one

**Pull request closed without merging**